static/app/views/organizationGroupDetails/groupDetails.tsx

```diff
@@ -415,14 +420,14 @@ class GroupDetails extends Component<Props, State> {
         eventView.getEventsAPIPayload(location)
       );
 
-      const replaysCount = data.data[0]['count()'].toString();
-
+      const ids = data.data.map(record => String(record.replayId));
```

Comment:
You could add a `count()` field and backend will group by `replayId`, so you'll have unique replayIds: https://sentry.io/organizations/sentry/discover/results/?field=replayId&field=count%28%29&name=RequestError%3A+GET+%22%2Fissues%2F3558727218%2Fevents%2Flatest%2F%22+200&project=11276&query=issue.id%3A3472704105+%21replayId%3A%22%22&sort=-replayId&statsPeriod=14d&yAxis=count%28%29

Reply:
oh, nice!